moped-database/metadata/tables.yaml

```diff
@@ -142,11 +142,9 @@
   delete_permissions:
     - role: moped-admin
       permission:
-        backend_only: false
```

Comment:
i have no idea why these changed

Reply:
oh maybe its because we updated the hasura version?

Reply:
I think that you've got it. Seems harmless and TIL - https://hasura.io/docs/latest/auth/authorization/permission-rules/#backend-only-permissions